Piwo.class.php

```diff
@@ -7,29 +10,55 @@ public static function onParserSetup( &$parser ) {
 		//the "python" magic word with execPy()
 		$parser->setFunctionHook( 'piwo', 'Piwo::execPy', Parser::SFH_OBJECT_ARGS );
 	}
+
+  /**
+   * write the given content to the given filePath
+   */
+  public static function writeStringToFile($filePath,$content) {
+		$f = fopen($filePath, "w");
+		fwrite( $f, $content );
+		fclose( $f );
+		unset($f);
+  }
+
 	// Render the output of {{#python:gram}}.
 	public static function execPy( $parser, $frame, $params ) {
 		//The inputs should contain a gram name and sys.argv
 		//The output should also be wikitext.
 		$page = WikiPage::factory( Title::newFromText( 'Gram:' . $frame->expand( $params[0] ) ) );
 		$name = $frame->expand( $params[0] );
+		$base="/tmp/";
+	  $pyCode= $base . $name . ".py";
+    $pyError= $base . $name . ".error";
```

Comment:
Very questionable code styling...
```suggestion
		$base = "/tmp/";
		$pyCode = $base . $name . ".py";
		$pyError = $base . $name . ".error";
```